src/api/createTaskLineModal.ts

```diff
@@ -0,0 +1,46 @@
+import type { App } from 'obsidian';
+import type { Task } from '../Task';
+
+/**
+ * Interface to remove all references to {TaskModal} in this file.
+ * This is necessary to make {createTaskLineModal} testable.
+ * Once Jest is configured to work with Svelte, this can be removed.
+ */
+export interface ITaskModal {
+    open(): void;
+}
+
+/**
+ * Signature of the factory method for {TaskModal}.
+ * This is necessary to make {createTaskLineModal} testable.
+ * Once Jest is configured to work with Svelte, this can be removed.
```

Comment:
What needs to happen to make Jest configured to work with Svelte? Could the comment describe that, so that we know what to do...

Reply:
Prior to refactoring the code I tried to configure Jest to work with Svelte. I wasn't successful, but then I have never used Svelte and I didn't try too hard. 

Is there somebody else who knows this? Maybe we create an issue and see if somebody steps in?

Reply:
> Prior to refactoring the code I tried to configure Jest to work with Svelte. I wasn't successful, but then I have never used Svelte and I didn't try too hard.

Ah right, thanks.

> Is there somebody else who knows this? Maybe we create an issue and see if somebody steps in?

If you had time to create an issue, that would help.